enterprise_access/apps/subsidy_request/models.py

```diff
@@ -0,0 +1,144 @@
+""" Models for subsidy_request. """
+
+from uuid import uuid4
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+
+class SubsidyRequest(TimeStampedModel):
+    """
+    Stores information related to a request for a subsidy (license or coupon).
+
+    .. no_pii: This model has no PII
+    """
+
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid4,
+        editable=False,
+        unique=True,
+    )
+
+    lms_user_id = models.IntegerField()
```

Comment:
we're gonna store this on the User model i think too. i think it's fine to store it here, but for the sake of saying it outloud: would we want to have a foreign key to the User here instead?

although, the User object would not be available if a particular user never authenticated against enterprise-access though (via any REST api), so may not be appropriate 🤔 -- lemme know what you think

Reply:
Yeah I don't think we should link it to a user object for the reason you mentioned.